hudi-common/src/main/java/org/apache/hudi/common/config/HoodieConfig.java

```diff
@@ -113,7 +113,7 @@ public <T> void setDefaultValue(ConfigProperty<T> configProperty, T defaultVal)
     }
   }
 
-  public Boolean contains(String key) {
+  public boolean contains(String key) {
```

Comment:
`containsKey` returns a primitive so this is just an opportunistic change to avoid auto-boxing where it is not required.